### PR TITLE
Merge 1.0 into 2.0

### PR DESCRIPTION
**Pull request by Doodoune**

### CODE REVIEW

File: CHANGELOG-1.0.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# 1.0.x
+
+## Bug Fixes
+
+- API-599: fix const not supported by PHP 5.6
```
Comment (LaurentPetard) [inline CHANGELOG-1.0.md]: Shouldn't we first tag the 1.0.2 before merging 1.0 into 2.0 ?

File: CHANGELOG-1.0.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+# 1.0.1 (2018-04-05)
```
Comment (LaurentPetard) [inline CHANGELOG-1.0.md]: In the 2.0 I did a single CHANGELOG. I think it's not necessary to do as the PIM and do a changelog for each version.

File: CHANGELOG-1.0.md (hunk 1 above)
Comment (Doodoune) [reply CHANGELOG-1.0.md]: we can merge before tag - it's not important no ?

Comment (LaurentPetard) [reply CHANGELOG-1.0.md]: Yes you're right ^^

**Pull request merged**